snippets/binarySearch.md

```diff
@@ -0,0 +1,22 @@
+### binarySearch
+
+Use recursion to perform a logarithmic search similar to `.indexOf()` that finds the index of a value within an array. The two differences being 
+1. Operation only works with sorted arrays
+2. Offers a major performance boost when compared to a linear search or `.indexOf()` 
+
```

Comment:
You need to add an explanation on how the snippet works. Remember to say at the very top `Uses recursion.` and then explain the rest of the code. It's pretty simple, but it needs to be explained for readers to grasp.